environment.yml

```diff
@@ -50,12 +49,12 @@ dependencies:
   - progressbar2
   - prov
   - psyplot >=1.5.0
-  - psy-maps >=1.5.0
-  - psy-reg >=1.5.0
-  - psy-simple >=1.5.0
+  # - psy-maps >=1.5.0  # nay Py3.13 yet
+  # - psy-reg >=1.5.0  # nay Py3.13 yet
+  # - psy-simple >=1.5.0  # nay Py3.13 yet
   - pyproj >=2.1
-  - pys2index  # only from conda-forge
-  - python >=3.10,<3.13
+  # - pys2index  # only from conda-forge  # not py313 compat
```

Comment:
It looks like this is solved, so now we're stuck on the psy-* packages and an unmaintained Julia installation, is that right?

Reply:
Or is NCL also an issue?

Reply:
yes and yes, and no - technically NCL is not a problem, but it brings down a bunch of deps to old versions (like esmpy/mf) so that may cause us headaches very soon, let me revive this PR now, bud :beer:

Reply:
well poop - now the env fails to solve again, at least this time it offers a clean fail for 3.13 - back to the drawing board see what's causing it :man_facepalming:

Reply:
I'll be a monkey's uncle :monkey_face: Mamba is indeed correct:
```yaml
---
name: testenv13
channels:
  - conda-forge
  - nodefaults

dependencies:
  - python=3.13
  - cartopy
  - esmvalcore 2.12*
  - iris >=3.11
  - shapely >=2.0.2
 ```

indeed fails to solve - what a PITA - I checked each of these deps and they should be fine individually wrt Python 3.13, time to get some butts kicked then :kick_scooter:

Reply:
got it! It's something in esmvalcore that's preventing that mini-env to solve

Reply:
well this is the one `- python >={{ python_min }},<3.13` - how the heck did this go through? Passed my review no less, but I suspect this is from a subsequent auto conda build :angry: